health-services/geopode-adapter/src/main/resources/Dockerfile

```
@@ -0,0 +1,20 @@
+#FROM egovio/alpine-maven-builder-jdk-8:1-master-NA-6036091e AS build
+FROM egovio/amazoncorretto:17-alpine3.19 AS build
+ARG WORK_DIR
+WORKDIR /app
+# Install Maven
+RUN apk add --no-cache maven
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)